packages/xo-server/src/api/host.js

```diff
@@ -23,6 +24,7 @@ set.description = 'changes the properties of an host'
 
 set.params = {
   id: { type: 'string' },
+  iscsiIqn: { type: 'string', optional: true },
```

Comment:
Can this value be removed from a host?

Reply:
Yes.

Reply:
How?

Which value is sent to the server to indicate that the field `iscsi_iqn` should be removed?

Reply:
I don't know if I understood your question but we send `null` to the server to removed the  `iscsi_iqn` value.

Reply:
You mean removed  `iscsi_iqn` from `other_config` ?

Reply:
`null` is not a valid type for the `iscsiIqn` param.

Reply:
It's my fault. :/
To remove `iscsiIqn`:  we send  an empty string "".
[I thought it's null because I receive null  from `xo-server`](https://github.com/vatesfr/xen-orchestra/pull/4208/files#diff-cbe526e73142a277b38a628ece1c6b57R146)

Reply:
IMO `xo-server`'s API should be symmetric, either use `null` or `''` for no IQN but not both.